buzzard.py

```diff
@@ -585,7 +586,12 @@ def drawSVG(svg_attributes, attributes, paths):
         out += "(footprint \"buzzardLabel\"\n" + \
             " (layer \"F.Cu\")\n" + \
             " (attr board_only exclude_from_pos_files exclude_from_bom)\n"
-
+    if args.outMode == "ki-5.1.x":
```

Comment:
Does this have to be so specific? Can it simply be 'ki5'? From my understanding this should work any version of KiCad 5.x.x

Reply:
I was not sure, so made it 5.1.x. I'll do the required change